assets/css/style.scss

```diff
@@ -1,7 +1,10 @@
-@use './modules/docs';
-@use './modules/base';
-@use './modules/genericons';
-@use './modules/menu';
-@use './modules/middleware';
-@use './modules/navbar';
-@use './modules/syntax';
+---
+---
```

Comment:
The triple equals is required by jekyll to make it parse a file. This is a bit weird for sure

Reply:
> The triple equals is required by jekyll to make it parse a file. This is a bit weird for sure

these are triple dashes! :eyeglasses:  ;)

Reply:
whoops